django_cloud_deploy/cloudlib/project.py

```diff
@@ -49,9 +51,11 @@ def __init__(self, cloudresourcemanager_service: discovery.Resource):
 
     @classmethod
     def from_credentials(cls, credentials: credentials.Credentials):
+        http_client = http.set_user_agent(http.build_http(), 'cloud-deploy')
```

Comment:
I think that 'django-cloud-deploy' might be better.

Reply:
si